Sources/Quick/Behavior.swift

```diff
@@ -4,7 +4,7 @@
 
 open class Behavior<Context> {
 
-    open static var name: String { return String(describing: self) }
+    public static var name: String { return String(describing: self) }
```

Comment:
This makes sense to me :+1:

But there is another possibility that `open class var` is the original intent. How do you think @Quick/core?